### PR TITLE
Created screens for feed page and post modal

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created modal component for the post
- feed page

Screenshot (156)

### CODE REVIEW

File: src/Pages/Profile/Profile.jsx [hunk 1]
```diff
@@ -39,12 +43,28 @@ const Profile = () => {
...
+                      <input className="input mt-2" type="text" placeholder="Write Caption"  />
+                      <textarea className="input mt-2" type="text" placeholder="What's Happening..."></textarea>
+
+                      <div className="flex gap-2 mt-2 mb-2">
+                        <input type="file" className="input file-upload w-3/4 cursor-pointer"/> <button className="btn w-3/4 text-caption">Post</button>
+                      </div>
+
+                </form>
+              </div>
+            </div> */}
```
Comment (gopalbbharadva) [inline src/Pages/Profile/Profile.jsx]: Remove comments if not needed.

**Review by gopalbbharadva — APPROVED** [review]
UI looks clean, Good to go.

File: src/Pages/AuthPage/Style.css [hunk 2]
```diff
@@ -3,7 +3,7 @@
      padding: .8rem 1rem;
      border-radius: 2px;
      outline: none;
-     background-color: var(--lite-blue);
+     /* background-color: var(--lite-blue); */
```
Comment (kiranpratyush) [inline src/Pages/AuthPage/Style.css]: We can remove comment if not necessary

**Review by kiranpratyush — APPROVED** [review]
The code looks clean. We can use prettier to format if not done

File: src/Components/PostCard/PostCard.jsx [hunk 3]
```diff
@@ -1,51 +1,49 @@
...
-                <img
-                  src="https://avatars.githubusercontent.com/u/84632214?v=4"
-                  className=""
-                  alt="user-name"
-                />
-              </div>
+      <div className="post-thumbnail-wrapper my-2 p-0 ">
+        <img
+          src="https://avatars.githubusercontent.com/u/84632214?v=4"
+          className=""
```
Comment (KoushikKintali) [inline src/Components/PostCard/PostCard.jsx:24]: if className is not required, u can remove that

File: src/Pages/Profile/Profile.jsx [hunk 4]
```diff
@@ -39,12 +43,28 @@ const Profile = () => {
...
           <div className="post-wrapper my-6 p-3">
             <PostCard /> 
             <PostCard />
             <PostCard />
           </div>
+
+          
+             {/* <div className="post-form-modal fixed top-0 left-0 right-0 bottom-0 flex justify-center items-start w-full ">
+              <div className="post-form-wrapper mx-3">
+                <p className="p-6 border-b-2"><FaIcons.FaTimes className="icons"/></p>
```
Comment (KoushikKintali) [inline src/Pages/Profile/Profile.jsx]: remove the commented code if not needed

**Review by KoushikKintali — APPROVED** [review]

File: src/Components/PostCard/PostCard.jsx (hunk 3 above)
Comment (pritam-kr) [reply src/Components/PostCard/PostCard.jsx:24]: ok

File: src/Pages/Profile/Profile.jsx (hunk 4 above)
Comment (pritam-kr) [reply src/Pages/Profile/Profile.jsx]: ok

File: src/Pages/Profile/Profile.jsx (hunk 1 above)
Comment (pritam-kr) [reply src/Pages/Profile/Profile.jsx]: its for modal

**Comment by pritam-kr** [comment]
Created three modal
- post, followers, following 
- to operate these modal created context for this

newwww

**Pull request merged**